### DISCUSSION TITLE
GFF3 track render order

### DISCUSSION THREAD

**Discussion by yonglbl** [GMOD/jbrowse-components#4311] [Q&A]
wondering how the order of GFF3 track got rendered, Is it based on the order listed on the GFF3 file? 
for example, my file looks like this:
gene1            ================================    ID=gene1
mRNA1            ================================    ID=mRNA1;
mRNA2            ================================    ID=mRNA2                  

will the browser render following the order on this file, first row gene1, second row mRNA1, then  mRNA2 ?

**Comment by cmdcolin** [comment]
this should be the case. it will, at the very least, reflect the order of the transcripts in the feature details panel. we have an open issue to add per-transcript labels to the gene glyph that i'd like to get added also to ensure clarity on this
